config/types/etcd.go

```diff
@@ -32,7 +33,9 @@ var (
 )
 
 // Options can be the options for any Etcd version
-type Options interface{}
+type Options interface {
```

Comment:
That doesn't seem right. The validation code already has this interface declared and uses that to do type-casts.